avm/res/automation/automation-account/powershell72-modules/main.bicep

```diff
@@ -0,0 +1,50 @@
+metadata name = 'Automation Account Powershell72 Modules'
+metadata description = 'This module deploys a Powershell72 Module in the Automation Account.'
+metadata owner = 'Azure/module-maintainers'
```

Comment:
Hey @ChrisLGardner,
please make sure to do the following things when you get the chance (in this order):
1. Deactivate actions in your fork
2. Afterwards, sync the latest changes from upstream 
3. Reactivate the actions again (if you wouldn't have deactivated them first, it would have triggered all workflows)
4. Merge the latest main branch now into your dev branch

Once you do, you'll also be exposed to the latest static tests which - in this case - would've failed on a few occaccsions. In this case specifically, please remove `metadata owner = 'Azure/module-maintainers'` from all new modules. Thank you (:

Reply:
Done.